### ISSUE TITLE
[Feature]: Tap to dismiss

### ISSUE THREAD

**Issue by ministryofsillywalks** [idaho/hassio-trash-card#154]
### Requirements

- [X] I have updated Mushroom to the latest available version
- [X] I did a search to see if there is a similar issue or if a pull request is open.

### Is your feature request related to a problem?

Currently, tapping the card does nothing. I would suggest a feature that when you tap the card, it gets dismissed.
The idea being, that once I take the trash out, I don't need to see the card anymore as a reminder. A quick tap on the card could dismiss it so that it doesn't block the dashboard. 

### Describe the solution you'd like

Tap the active card to dismiss it and hide it from view.

**Comment by idaho** [comment]
Good idea.
In my opinion it would make sense that the event will disappeare then in all browsers/apps.
Or what did you think?

**Comment by ministryofsillywalks** [comment]
Yes I would agree that it should disappear on all browsers/apps.
Maybe link it to a conditional helper to give more flexibility in toggling it. I know of people who track their trashcans presence via ble trackers. Having the bin go out of range (at the road) could this toggle the helper and make the card disappear.

**Comment by Count0Int** [comment]
Agreed,  this would be very useful. 

Still have to ask my son if he took out the garbage/recycling. This would avoid that.

**Comment by Arcane1980** [comment]
@idaho I am also seeking this. I have seen someone else ask for the same thing in different words.

The card is awesome. I love the way it displays and is a game changer for me and my memory issues.

Being able however to tap the icon for the "Organics Bin" to show that I took it out already would be just that next level. I notice that for others it's a matter of either the wife continually using the icon as an excuse to keep nagging the husband, or the same from parents to kids. FOR ME it's about the fact that my short term memory is damaged from trauma and I will regularly try and take the bins out 3 or 4 times forgetting I already did it.

BEFORE I found trashcard, I would just not remember to do them at all!

Being able to have trashcard show the next scheduled events until I click them would just make my life so much simpler and help rectify a neurological issue that I unfortunately don't have a wife and kids around to use as reminders.

In addition to this, it would be nice to know how to make the card only visible if there are reminders to show (ie on my home it leaves a small space/place holder when there are no reminders - I'd prefer the card disappear all together when nothing to show), or "No reminders" so that at least it's not a blank spot.

Thanks in advance @idaho , the card is great already, this will just take my own life to the next level ;)

**Comment by jjindrich** [comment]
Fully agree. Maybe there is possible solution to configure Visibility conditions. For now I configured condition to display it when Sun goes down.
But it will be great to click on something and hide icon. Any idea how to automate it?

**Comment by HyperCriSiS** [comment]
It should be with long click not normal click to avoid accidental clicks 😅

**Comment by priyam1309** [comment]
I agree this is very useful feature. Since its a reminder and reminders are supposed to be completed.

**Comment by AlexJacu** [comment]
This is be an additional option to request. 
Then it should be possible a use tab or double tap to invoke an action
https://github.com/idaho/hassio-trash-card/issues/584
Add tap-action for cards to open other page #584
